src/go.mod

```diff
@@ -18,15 +19,16 @@ require (
 )
 
 require (
+	github.com/dgraph-io/badger v1.6.0 // indirect
+	github.com/etcd-io/bbolt v1.3.3 // indirect
+	github.com/gavv/httpexpect v2.0.0+incompatible // indirect
 	github.com/magefile/mage v1.10.0 // indirect
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/shurcooL/sanitized_anchor_name v1.0.0 // indirect
 	google.golang.org/protobuf v1.27.1 // indirect
-	github.com/dgraph-io/badger v1.6.0 // indirect
-	github.com/etcd-io/bbolt v1.3.3 // indirect
-	github.com/gavv/httpexpect v2.0.0+incompatible // indirect
 )
 
 replace (
+	github.com/ODIM-Project/ODIM/lib-dmtf => ../lib-dmtf
```

Comment:
Don't we need one more replace? for lib-persistece-manager?

Reply:
It is added here #43